docs/developers/define-secrets.rst

```diff
@@ -305,6 +321,9 @@ Change the field type to password so that the value isn't displayed any time som
 Do not use sections.
 Phalanx requires all of the secret entries be top-level fields outside of any section.
 
+Newlines will be converted to spaces when pasting the secret value.
+If newlines need to be preserved, be sure to mark the secret with ``onepassword.encoded`` set to ``true`` in :file:`secrets.yaml`, and then encode the secret in base64 before pasting it into 1Password.
```

Comment:
Maybe include a example command that shows piping a file through base64 to encode a file?